news/_posts/2016-09-04-kernel-release-cycle-15.md

```diff
@@ -0,0 +1,43 @@
+---
+author: "@dlech"
+title: "Kernel Release Cycle 15"
+---
+
+This release is all about I2C. I2C (and sound) was broken on the EV3 if you were
+running `flash-kernel 3.35ev3dev10` or higher. Actually it has been broken since
+the very first days of ev3dev. It was just by chance that they worked in previous
+releases because of a strange interaction with the serial port on input port 1.
+
+Also, we now have I2C support on FatcatLab's EVB for BeagleBone. Due to a hardware
+shortcoming (no GPIO on input port pin 2), not all I2C sensors may be automatically
+detected, but many will be. Also, due to another hardware shortcoming (no 9V on
+input port pin 1), the NXT Ultrasonic sensor may not work reliably even though
+it is detected.
+
+<!--more-->
+
+We also enable some MIDI drivers in the EV3 kernel, so you can try hooking it up
+to your MIDI keyboard or see what happens with @JorgePe's [laser harp].
+
+[laser harp]: http://ofalcao.pt/blog/series/lego-laser-harp
+
+In this round of releases, we have:
+
+* `v4.4.19-15-ev3dev-ev3` for EV3.
+* `v4.4.19-ti-rt-r41-15-ev3dev-bb.org` for BeagleBone.
+* `v4.4.19-15-ev3dev-rpi` for Raspberry Pi 0/1.
+* `v4.4.19-15-ev3dev-rpi2` for Raspberry Pi 2/3.
+
+You can also find this kernel in our latest [nightly build][download].
+
+[download]: https://oss.jfrog.org/list/oss-snapshot-local/org/ev3dev/brickstrap/
```

Comment:
Because the news feed might be seen by people at a time when more builds have been published, I think we should start including the image names somewhere here or the date for the folder which the builds are in.